### PR TITLE
8282766: Create release notes for JavaFX 18

### PR DESCRIPTION
**Pull request by abhinayagarwal**
Add release notes for JavaFX 18 to the repository

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8282766](https://bugs.openjdk.java.net/browse/JDK-8282766): Create release notes for JavaFX 18

### Reviewers
 * [Kevin Rushforth](https://openjdk.java.net/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Johan Vos](https://openjdk.java.net/census#jvos) (@johanvos - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jfx pull/752/head:pull/752` \
`$ git checkout pull/752`

Update a local copy of the PR: \
`$ git checkout pull/752` \
`$ git pull https://git.openjdk.java.net/jfx pull/752/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 752`

View PR using the GUI difftool: \
`$ git pr show -t 752`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jfx/pull/752.diff">https://git.openjdk.java.net/jfx/pull/752.diff</a>

</details>

### CODE REVIEW

File: doc-files/release-notes-18.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+## Important Changes
+
+### Deprecate JavaFX GTK 2 library for removal
+
+The JavaFX GTK 2 library is deprecated and will be removed in a future release. The JavaFX runtime issues a warning if the GTK 2 library is requested on the command line via `java -Djdk.gtk.version=2`.
+The JavaFX runtime also issues a warning if the GTK 2 library is selected as a fallback, which happens if the GTK 3 library cannot be loaded. Application developers should avoid requesting the GTK 2 library.
+
+See [JDK-8273089](https://bugs.openjdk.java.net/browse/JDK-8273089) for more information.
+
+## List of Enhancement
```
Comment (kevinrushforth) [inline doc-files/release-notes-18.md:18]: That should be "Enhancements" (plural).

**Review by kevinrushforth — COMMENTED** [review]
Looks good other than one typo: Enhancement --> Enhancements

**Review by kevinrushforth — CHANGES_REQUESTED** [review]
I see that this is targeted to the `master` branch. It should be targeted to `jfx18`. Since your branch is based on `master` you will need to rebase your branch and force-push before changing the target of the PR to `jfx18`.

**Review by kevinrushforth — APPROVED** [review]

**Comment by abhinayagarwal** [comment]
@kevinrushforth Before I integrate the PR, can you have a look at [my comment on the issue](https://bugs.openjdk.java.net/browse/JDK-8282766?focusedCommentId=14482267&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14482267)?

**Comment by kevinrushforth** [comment]
> @kevinrushforth Before I integrate the PR, can you have a look at [my comment on the issue](https://bugs.openjdk.java.net/browse/JDK-8282766?focusedCommentId=14482267&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14482267)?

Oh, I had missed your question.. Yes, those three issues were intentionally added to the JBS filter. Even though they are `noreg-build` issues, they touch on behaviors that are visible to developers.

File: doc-files/release-notes-18.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+[JDK-8160597](https://bugs.openjdk.java.net/browse/JDK-8160597)|IllegalArgumentException when we initiate drag on Image|window-toolkit
+[JDK-8227371](https://bugs.openjdk.java.net/browse/JDK-8227371)|Drag&Drop while holding the CMD key does not work on macOS|window-toolkit
+[JDK-8242544](https://bugs.openjdk.java.net/browse/JDK-8242544)|CMD+ENTER key event crashes the application when invoked on dialog|window-toolkit
+[JDK-8269967](https://bugs.openjdk.java.net/browse/JDK-8269967)|JavaFX should fail fast on macOS below minimum version|window-toolkit
+[JDK-8269968](https://bugs.openjdk.java.net/browse/JDK-8269968)|[REDO] Bump minimum version of macOS for x64 to 10.12|window-toolkit
+[JDK-8271398](https://bugs.openjdk.java.net/browse/JDK-8271398)|GTK3 drag view image swaps red and blue color channels|window-toolkit
+[JDK-8274929](https://bugs.openjdk.java.net/browse/JDK-8274929)|Crash while reading specific clipboard content|window-toolkit
+[JDK-8275723](https://bugs.openjdk.java.net/browse/JDK-8275723)|Crash on macOS 12 in GlassRunnable::dealloc|window-toolkit
+
```
Comment (tsayao) [inline doc-files/release-notes-18.md:105]: [REDO] Wrong stage gets focused after modal stage creation https://bugs.openjdk.java.net/browse/JDK-8271054 ?

Comment (kevinrushforth) [reply doc-files/release-notes-18.md:105]: No, that fix is in 19. These release notes are for 18.

Comment (tsayao) [reply doc-files/release-notes-18.md:105]: Sorry, my mistake.

**Review by johanvos — APPROVED** [review]

**Comment by abhinayagarwal** [comment]
@kevinrushforth What about the "noreg-doc" and "test_sprint"  issues in the filter?

**Comment by kevinrushforth** [comment]
@abhinayagarwal  I would expect the `noreg-doc` fixes to be included if they touch user-visible documentation. I see the two you mentioned in the filter I provided, although I don't see them in this PR. As for `test_sprint`, the filter doesn't consider that label. If they are test bugs then they should have `testbug` and/or `noreg-self` and be excluded as a result.

**Comment by kevinrushforth** [comment]
However, the two you mentioned are sort of borderline as to whether you want to include it or not. I would be just as happy excluding them.

**Pull request closed without merging**